domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimData.java

```diff
@@ -55,8 +54,6 @@
     @NotNull
     private final Amount amount;
 
-    @NotNull
-    @Min(0)
```

Comment:
Don't we still want the `Min(0)`? I think it won't activate as validation if the value is `null`

Reply:
That is never going to be invoked as `feeAmountInPennies` will never be passed from front-end after IOC. will always be initiated in callback now. Anyways added that back for current scenario.